python-sdk/pyproject.toml

```diff
@@ -87,8 +87,11 @@ ftp = [
 ]
 openlineage = ["openlineage-airflow>=0.17.0"]
 
-databricks = ["databricks-cli",
-    "apache-airflow-providers-databricks"]
+databricks = [
+    "databricks-cli",
+    "apache-airflow-providers-databricks",
+    "databricks-sql-connector<2.9.0",
```

Comment:
This temporary requirement since https://github.com/databricks/databricks-sql-python/releases/tag/v2.9.0 is breaking our test. they have PR https://github.com/databricks/databricks-sql-python/pull/191 to fix it but we will have to wait for release

Reply:
Could have been a separate PR, might be easy to revert :)

Reply:
done https://github.com/astronomer/astro-sdk/pull/2013